packages/server/lib/modes/run.js

```diff
@@ -1105,6 +1111,16 @@ module.exports = {
       return this.currentSetScreenshotMetadata(data)
     }
 
+    if (options.experimentalSingleTabRunMode && options.testingType === 'component' && !options.isFirstSpec) {
+      // reset browser state to match e2e behavior when opening/closing a new tab
+      return openProject.resetBrowserState().then(() => {
```

Comment:
We reset the state here, much like we do between tabs, so this minimizes the chance of negatively impacting spec isolation.

Reply:
maybe this doesn't matter since this is experimental, but if something goes wrong, this loses the test retries for CT.

Reply:
Oh true, this seems like a problem - something we will definitely need to highlight in the notes, unless we can find a fix prior to releasing this experiment (if that's even acceptable, will need to update the tech brief with this drawback... good point).

Reply:
Actually @emilyrohrbough I don't get it - I tried to use `retries` but it works okay: https://github.com/cypress-io/cypress/pull/23104/commits/4c99b4d380725d913db7a3e005d0ff77e8c34644#diff-ff7a203eea50ba8d34ac184c54310e07e4f279447a923ba9bec878e643b9e289R699-R700

Is there another way to configuring this or an edge case that I'm missing?

Reply:
The retries are only for the initial browser launch right?  I'm not sure we'd need retries for when we're navigating in this new way